modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceJWTGrantTestCase.java

```diff
@@ -167,7 +167,7 @@ public void testPasswordGrantBasedSelfContainedAccessTokenGeneration()
         jwtAssertion = oidcTokens.getIDTokenString();
         alias = oidcTokens.getIDToken().getJWTClaimsSet().getAudience().get(0);
         issuer = oidcTokens.getIDToken().getJWTClaimsSet().getIssuer();
-        Assert.assertEquals(oidcTokens.getIDToken().getJWTClaimsSet().getClaim(COUNTRY_OIDC_CLAIM), COUNTRY_CLAIM_VALUE,
+        Assert.assertNull(oidcTokens.getIDToken().getJWTClaimsSet().getClaim(COUNTRY_OIDC_CLAIM),
                 "Requested user claims is not returned back in self contained access token based on password "
```

Comment:
Error message needs to be improved. 

E.g. "Requested user claims contains claims not defined in requested scopes in self contained access token based on password claim"